### ISSUE TITLE
Charred Trees missing textures 1.18.2

### ISSUE THREAD

**Issue by Shean000** [MysticMods/MysticalWorld#320]
Hello, my issue is sort of similar to  [1.16.5] Can't disable charred tree. #310
The reason I would want to keep the trees from spawning is that the charred trees have purple/black boxes as a texture.
I already contacted the guys from dynamic trees on the discord and they are looking into being able to keep trees from spawning individually.
However they said it would be better to fix the underlying issue of the missing texture.
I am using a big modpack including dynamic trees so i am not sure if anything else could cause this issue.
![Képernyőkép 2022-07-12 120403](https://user-images.githubusercontent.com/109160068/178519990-f432a0b9-1897-4a4d-870a-5e821252199d.png)

**Comment by Harleyoc1** [comment]
@noobanidus 
The block state and model files here aren't being read because the blocks are now using the `dynamictrees` namespace rather than `mysticalworld` because of the omission of [this line](https://github.com/MysticMods/MysticalWorld/blob/db5ad5d2a600492920789843da23b0e599f589c0/src/main/java/mysticmods/mysticalworld/MysticalWorld.java#L104) after porting to 1.18. I'm guessing that's because you ported before we did. 

Adding back this line should fix this issue.

**Comment by noobanidus** [comment]
> @noobanidus The block state and model files here aren't being read because the blocks are now using the `dynamictrees` namespace rather than `mysticalworld` because of the omission of [this line](https://github.com/MysticMods/MysticalWorld/blob/db5ad5d2a600492920789843da23b0e599f589c0/src/main/java/mysticmods/mysticalworld/MysticalWorld.java#L104) after porting to 1.18. I'm guessing that's because you ported before we did.
> 
> Adding back this line should fix this issue.

Yeah, I completed the port several months ago, I just only released it recently.
